templates/swifty-gtk4/template.json

```diff
@@ -0,0 +1,33 @@
+{
+  "description": "An opinionated Gtk4 Application Template.",
+  "variables": {
+    "PROGRAM_NAME": {
+      "summary": "The name of the program:",
```

Comment:
Make sure that these summaries are not capitalized and start with articles like "the" or "a", so that the wizard says the right thing. This summary should be, "the name of the program" because of how it's used:

https://github.com/Prince781/valdo/blob/f21ef7b9bb2a6e43795565c76f076b1eeb9fced8/src/main.vala#L114